paddle/operators/detail/grpc_server.cc

```diff
@@ -184,15 +183,20 @@ void AsyncGRPCServer::HandleRequest(bool wait, grpc::ServerCompletionQueue* cq,
       break;
     }
 
+    assert(tag);
     if (wait && !done_) {
       Wait();
     }
 
     RequestBase* base = (RequestBase*)tag;
+    // reference:
+    // https://github.com/tensorflow/tensorflow/issues/5596
+    // https://groups.google.com/forum/#!topic/grpc-io/xftlRy-IQwM
     if (!ok) {
-      VLOG(4) << cq_name << " recv no regular event";
+      LOG(WARNING) << cq_name << " recv no regular event:argument name"
+                   << base->GetReqName();
+      // FIXME(gongwb): delete the old one?
       TryToRegisterNewOne();
-      delete base;
```

Comment:
No other places to release this memory then.

Reply:
So add FixME.

Reply:
Done.

Reply:
I'm not sure it's a grpc bug or it's our application bug.
When delete base often, I met an [error](https://github.com/grpc/grpc/issues/13983)

Reply:
So add `FixMe` here.